src/styles/block/cards.scss

```
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 48px 46px 0;
+  padding: 32px 16px 0;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $background_button-color;
+  
+  &__img {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  
+  &__product-name {
+    @extend %product-name-size_font;
+    margin: 0 0 4px;
+    color: $text-accent-color;
+  }
+  
+  &__product-code {
+    @extend %code-and-count-size_font;
+    align-self: start;
+    color: $text-accent-color;
+    font-style: initial;
+    margin: 0 0 16px;
+  }
+  
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 0 0 24px;
+  }
+  
+  &__review-count {
+    @extend %code-and-count-size_font;
+    margin: 0;
+    text-align: right;
+    color: $text-accent-color;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+  }
+  
+  &__price-text {
+    @extend %price-text-size_font;
+    color: $text-secondary-color;
+    margin: 0;
+  }
+  
+  &__price-value {
+    @extend %price-value-size_font;
+    color: $text-accent-color;
+    font-weight: bold;
+    margin: 0;
+  }
+  
+  &__button {
+    @extend %button-size_font;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $main-color;
+    height: 40px;
+    border: 1px solid $main-color;
+    border-radius: 5px;
+    margin: 0 0 16px;
+    color: $background_button-color;
+    cursor: pointer;
+    text-decoration: none;
+  }
+  
+  &__button:hover {
+    border-color: 1px solid $main-color;
+    background-color: $background_button-color;
+    color: $main-color;
+  }
+}
+
+.card:nth-child(4n) {
+  margin-right: 0;
+}
```

you can add the style inside main `.card` block
```suggestion
  }
  
  &:nth-child(4n) {
    margin-right: 0;
  }
}

```